src/test/java/uk/gov/hmcts/reform/pcqloader/helper/PayloadMappingHelperTest.java

```diff
@@ -549,6 +549,11 @@ private static PcqMetaData jsonMetaDataObjectFromString(String jsonString) throw
         return new ObjectMapper().readValue(jsonString, PcqMetaData.class);
     }
 
-
+    private byte[] decodeFromString(String src) {
+        if (src.isEmpty()) {
```

Comment:
What about src null?

Reply:
Checked with StringUtils for null and empty both